volatility3/framework/plugins/windows/dumpfiles.py

```diff
@@ -355,10 +358,14 @@ def run(self):
         ):
             raise ValueError("Cannot use filter flag with an address flag")
 
-        if self.config.get("virtaddr", None) is not None:
-            offsets.append((self.config["virtaddr"], True))
-        elif self.config.get("physaddr", None) is not None:
-            offsets.append((self.config["physaddr"], False))
+        if self.config.get("virtaddr"):
+            for virtaddr in self.config["virtaddr"]:
+                offsets.append((virtaddr, True))
+
+        elif self.config.get("physaddr"):
+            for physaddr in self.config["physaddr"]:
+                offsets.append((physaddr, False))
```

Comment:
If the physical and virtual offsets all go into the same list, how do we ensure the offset found is from the right layer?  Something seems... off... about this plugin?

Reply:
Oh, I see, we're storing a list of tuples...  5:S

Reply:
Also, this `elif` means you'll only ever get one or the other in the list...  5:S

Reply:
Accepted both and then get the PIDs otherwise.

Reply:
Well, it accepts virtual, then it accepts physical if no virtual, then it tries pids.  If you define both, virtual will take precedence (because it's `elif`, not just `if`)

Reply:
The `else` then porbably wants to be a guard on both configs not being specified...

Reply:
Hey sorry, I was making comments while prepping the push to make sure I covered everything